### PR TITLE
`pip install` packaging support

### LINKED ISSUE #23: Upload to PyPI
Any chance of adding this project to [PyPI](https://pypi.python.org/pypi)? I'd rather `pip install pmcli` then worry about dependencies manually.

There's a pretty good how-to here:
https://packaging.python.org/distributing/

### PR DESCRIPTION
**Pull request by matteing**
Closes #23 

I was able to package the files into a wheel format. This will be a slightly breaking change, as many folders did move around. However, old installation method should work just fine.

If there's any bugs, comment and I'll hunt 'em down. :smile:

```
$ pip3 install gpymusic
$ gpymusic-setup
$ gpymusic
```

Also, send me an email at sergiomattei[at]outlook.com so we can sort out the package ownership on PyPI :p I can transfer the package to you.

EDIT: Currently fixing a critical bug. 
EDIT2: Fixed. Working like a charm :smile:

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+appdirs==1.4.3
```
Comment (christopher-dG) [inline requirements.txt:1]: Is this file automatically generated? If not, I'm wondering if listing dependencies of packages that are already listed in here are necessary.

File: gpymusic/main.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 #! /usr/bin/env python3
```
Comment (christopher-dG) [inline gpymusic/main.py]: Is this file still necessary now that it's duplicated in `bin`?

File: requirements.txt (hunk 1 above)
Comment (matteing) [reply requirements.txt:1]: pip freeze > requirements.txt.
Allows for more reproducible environments by locking versions and so any breaking change upstream doesn't break this immediately.

Yes, auto generated.

File: gpymusic/main.py (hunk 2 above)
Comment (matteing) [reply gpymusic/main.py]: Kept it there just in case. It can be deleted for sure.

**Comment by christopher-dG** [comment]
I was hoping to have enough time to properly look over everything and make the small changes I wanted this weekend, but it looks like I'll probably have to wait until next weekend. I can assure you that this will (eventually) get merged!

**Comment by matteing** [comment]
@christopher-dG Got it. No rush :smile:

**Comment by christopher-dG** [comment]
Okay, I think this is ready to go! I sent you an email.

**Comment by christopher-dG** [comment]
Merged, thanks again :smile:
